cypress/src/components/CartContent.js

```diff
@@ -0,0 +1,8 @@
+/*globals cy*/
+class CartContent {
+    productIsVisible() {
+        cy.get('#tbodyid').should('exist')
```

Comment:
In some cases, elements might exist in the DOM structure but be hidden. Better to check if the element is visible.